ttnn/cpp/ttnn/operations/ccl/ccl_op_fusion.hpp

```diff
@@ -35,7 +36,8 @@ struct AllGatherFusedOpSignaler {
 
     void init_fused_op(
         const std::vector<CoreCoord>& fused_op_receiver_cores_noc,
-        const std::vector<uint32_t>& fused_op_receiver_signal_semaphores
+        const std::vector<uint32_t>& fused_op_receiver_signal_semaphores,
+        bool mcast_fused_op_cores = true
```

Comment:
Could this be const?

If possible it's good to keep all parameters const as much as possible.